src/index.js

```diff
@@ -1,182 +1,194 @@
-const dicomParser = require('dicom-parser')
-const asyncIterableToBuffer = require('./asyncIterableToBuffer')
-const getDataSet = require('./getDataSet')
-const JSONWriter = require('./JSONWriter')
-const InstanceDeduplicate = require('./InstanceDeduplicate')
-const DeduplicateWriter = require('./DeduplicateWriter')
-const ImageFrameWriter = require('./ImageFrameWriter');
-const CompleteStudyWriter = require('./CompleteStudyWriter');
-const IdCreator = require('./IdCreator');
-const fs = require('fs')
-const dirScanner = require('./dirScanner')
-const ScanStudy = require('./ScanStudy')
-const HashDataWriter = require('./HashDataWriter')
-const JSONReader = require('./JSONReader')
-const {getArg,hasArg, getRemainingArgs, showHelp} = require('./../src/args');
-const path = require('path');
-const homedir = require('os').homedir();
-const dicomwebDefaultDir = '~/dicomweb';
-const Stats = require('./stats');
-const VideoWriter = require('./VideoWriter')
+const dicomCodec = require("@cornerstonejs/dicom-codec");
```

Comment:
About this file entire change. 
Its not ideal having more changes than necessary (it also applies to this case: i.e lint), but until there is none quality code strategy for linting I strongly recommend ignoring if possible formatting changes on the review process.